SimPEG/MT/Utils/MT1Danalytic.py

```diff
@@ -0,0 +1,111 @@
+# Analytic solution of EM fields due to a plane wave
+
+import numpy as np, SimPEG as simpeg
+
+def getEHfields(m1d,sigma,freq,zd,scaleUD=True):
```

Comment:
another thing to chat about wrt analytics: I think we should try and have consistent inputs (where applicable). Most functions have at least some concept of a mesh, model, freq / time, so it would be good to try and use the same names / orders 
(I don't think you need to change anything for this pr though, just something to think about!)